examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -138,6 +138,33 @@ class GalleryAppState extends State<GalleryApp> {
       );
     }
 
+    final Map<String, WidgetBuilder> _kRoutes = new Map<String,
+      WidgetBuilder>.fromIterable(
+      // For a different example of how to set up an application routing table,
+      // consider the Stocks example:
+      // https://github.com/flutter/flutter/blob/master/examples/stocks/lib/main.dart
+      kAllGalleryItems,
+      key: (GalleryItem item) => item.routeName,
+      value: (GalleryItem item) =>
+        (BuildContext context) {
```

Comment:
If this was on the previous line the indents would be more flutter-regular.

Reply:
Done.